### PR TITLE
Add new error and context examples, add Fizz Buzz exercise.

### PR DESCRIPTION
**Pull request by witchard**

### CODE REVIEW

File: class.slide [hunk 1]
```diff
@@ -168,13 +162,29 @@ Go Overview
...
 .play -edit err.go /START FUNC OMIT/,/END FUNC OMIT/
 
+* Error Example - Wrapping
+.play -edit err2.go /START MAIN OMIT/,/END MAIN OMIT/
+
+* Documentation
+- Docs are never far from the source.
+- You can do the same.
+- By convention, start doc comments with the name of the function/type.
+.link https://golang.org/pkg/crypto/tls/#X509KeyPair LoadX509KeyPair() Example
```
Comment (witchard) [inline class.slide:180]: This was way later... but I thought it would be neater up here

File: class.slide [hunk 2]
```diff
@@ -336,8 +339,16 @@ Go Overview
 * Member function aren't just for structs
 .play -edit memberfunctions.go /START OMIT/,/END OMIT/
 
+* Exercise: Loprem Ipsum Generator
+- Create a Lorem Ipsum generator.
+- It should take one command line parameter for the number of paragraphs to generate.  See os.Args.
+- Read sample text from lorem_ipsum.txt into a slice.
+- The paragraphs should be printed in random order.  See rand.Intn().
+- It's ok to print the same paragraph more than once.
+- Extension: Count the unique words in Loreum Ipsum.
```
Comment (witchard) [inline class.slide:348]: Moved this after the map section to facilitate the extension exercise

**Review by witchard — COMMENTED** [review]
Added some inline comments to explain a few things

File: class.slide (hunk 1 above)
Comment (billallen256) [reply class.slide:180]: This is a good idea.  I might add a link to DevDocs as well.

**Pull request merged**